packages/terser/src/terser_minified.js

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env node
```

Comment:
please rename to the old path, or at least put the rename in its own commit so I don't have a giant block of "newly added" code that I can't tell what you changed

Reply:
Yeah that's fair, thought the naming of the files made more sense this way, but for the sake of actually able to code review it I'm happy flipping it around

Reply:
put the file rename in a seprate commit

Reply:
that didn't quite work because the first commit still references the new file, I wasn't able to just fetch and get the test failure off the first commit.

I don't even really agree with renaming the file. If there's only ever going to be one file in this package, why not name it index? it makes the rules shorter since there are no data deps

Reply:
> I wasn't able to just fetch and get the test failure off the first commit.

I have a couple of fixups that apply to the first commit, they'd need to be rebased in to allow for that.

I couldn't get `if(require.module === module) { main() }` to work here under the testing environment. It would still call the main method even when that file was being required and wasn't the entry point.
I'd prefer a single file aswell but I don't get how to make it work.

Reply:
you mean `if (require.main === module) {` ?
What didn't work about it? Let's please figure that out rather than introduce a refactoring purely as a workaround for a problem we don't understand

Reply:
The refactor is still required, if we can get `if (require.main === module) {` to work then that just removed the need for two files.

Reply:
ok tried it again and it's working now, not sure what happened before